src/components/Mission.js

```diff
@@ -0,0 +1,73 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMission, missionSpace } from '../redux/missions/missions';
+import './missions.css';
+
+const Mission = () => {
+  const dispatch = useDispatch();
+  const { missions, loading } = useSelector((state) => state.missions);
+  useEffect(() => {
+    dispatch(fetchMission());
+  }, [loading, dispatch]);
+
+  if (loading) {
+    return <h1> Loading...</h1>;
+  }
+
+  const jetclick = (id) => {
+    dispatch(missionSpace(id));
+  };
+
+  return (
+    <table className="missions-tab" id="tab">
+      <thead>
+        <tr>
```

Comment:
- [ ] This component is being rendered upon every visit and it's causing the joined mission to get lost because the page is being re-rendered upon every visit. It's supposed to fetch the data once and rendered, so when you visit another page and come back the joined mission stays as it was when you left. Kindly work on it so it's similar to the Rockets page. 👍 

![image](https://user-images.githubusercontent.com/50941074/205440165-587d1640-75b3-4c01-afca-eaa6e7b294be.png)